metriq_gym/dispatch_qv.py

```diff
@@ -31,11 +31,12 @@ def main():
 
         return 0
 
-    logging.info(f"Dispatched {args.trials} jobs")
```

Comment:
I can't leave a comment for this as this isn't in the diff edits, but the first line in this file for the comment that describes it should probably say something about QV (as does the CLOPS one).

Reply:
Done.